pkg/piecestore/psserver/psdb/psdb.go

```diff
@@ -205,6 +208,32 @@ func (db *DB) Migration() *migrate.Migration {
 					`CREATE INDEX IF NOT EXISTS idx_bwa_serial ON bandwidth_agreements (serial_num)`,
 				},
 			},
+			{
+				Description: "delete pieces that are stored on drive using 'ttl' old db",
```

Comment:
we are not using `ttl` table to remove pieces

Reply:
that was description.. saying that it is deleting pieces that were stored using ttl db. Please suggest the description. I am open  :-)

Reply:
`delete old pieces that are stored on drive` will be enough

Reply:
done

Reply:
I don't see this change.

Reply:
sorry I was addressing the comments as I was making the change.. lol 👍